Unity/Assets/GameSystems/Unity/GameObjects/ObstacleSpawner/PipeObstacle.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections;
+using UnityEngine;
+
+public class PipeObstacle : MonoBehaviour, ISpawn
+{
+    [SerializeField] SpriteRenderer spriteRenderer;
```

Comment:
Need a private after the [serializeField], as mentioned here - https://github.com/AdolfoPSakata/UnityTraining-2/pull/1